sdk/ai/azure-ai-inference/azure/ai/inference/models/__init__.py

```diff
@@ -14,13 +14,25 @@
 
 
 from ._models import (  # type: ignore
-    AudioContentItem,
+    AudioUrlContentItem,
     ChatChoice,
     ChatCompletions,
     ChatCompletionsNamedToolChoice,
     ChatCompletionsNamedToolChoiceFunction,
+    ChatCompletionsResponseFormat,
+    ChatCompletionsResponseFormatJsonObject,
+    ChatCompletionsResponseFormatJsonSchema,
+    ChatCompletionsResponseFormatText,
     ChatCompletionsToolCall,
     ChatCompletionsToolDefinition,
+    ChatMessageAudioDataContentItem,
+    ChatMessageInputAudioUrl,
+    ChatRequestAssistantMessage,
+    ChatRequestDeveloperMessage,
+    ChatRequestMessage,
+    ChatRequestSystemMessage,
+    ChatRequestToolMessage,
+    ChatRequestUserMessage,
```

Comment:
The ChatRequestXxxxMessage should not be here. TypeSpec should be marking them as internal. I hand-write them in the _patch.py file